packages/cli/lib/services/dryrun.service.ts

```diff
@@ -477,17 +477,16 @@ export class DryRunService {
                 return { success: false, error: new NangoError(content, 500), response: null };
             }
 
+            const filename = `${syncName}-${nangoProps.providerConfigKey}.js`;
             try {
-                const wrappedScript = `
-                    (function() {
-                        var module = { exports: {} };
-                        var exports = module.exports;
-                        ${script}
-                        return module.exports;
-                    })();
+                const wrappedCode = `(function() { var module = { exports: {} }; var exports = module.exports; ${script}
+                    return module.exports;
+                })();
```

Comment:
to make it a bit more readable we could also add a `// SCRIPT START` comment in the wrapped code and look for its line to calculate the offset. Up to you

Reply:
Yeah, I’m fine with this for now. In my initial implementation I tried to be clever and calculate the lines based on line breaks but this code isn’t complicated and also I feel like a random comment would be strange.